apps/analysis/models.py

```diff
@@ -0,0 +1,77 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField
+
+from user.models import User
+from project.models import Project
+from entry.models import Entry
+
+
+class Analysis(models.Model):
+    title = models.CharField(max_length=255)
+    team_lead = models.ForeignKey(
+        User,
+        blank=True,
+        null=True,
+        on_delete=models.CASCADE,
+    )
+    created_on = models.DateField(blank=True, null=True)
+    analysis_pillars = models.ManyToManyField(
+        'AnalysisPillar',
+        blank=True,
+    )
+
+    def __str__(self):
+        return self.title
+
+
+class AnalysisPillar(models.Model):
+    main_statement = models.TextField()
+    information_gap = models.TextField()
+    filters = JSONField(blank=True, null=True, default=None)
+    assignee = models.ForeignKey(
+        User,
+        blank=True,
+        null=True,
+        on_delete=models.CASCADE
+    )
+
+    def __str__(self):
+        return self.main_statement
```

Comment:
This might show long text in the admin panel. Can you check if it does?
If it does let's use `self.main_statement and self.main_statement[:255]` to avoid long text.